operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ Entries should include a reference to the Pull Request that introduced the chang
 
 ## main / unreleased
 
+* [BUGFIX] Added missing endpoint for OTLP in NGINX #3424
```

Comment:
[nit] We're use to reference the PR number. Also could you move this entry below, along with the other `[BUGFIX]` entries, please?
```suggestion
* [BUGFIX] Added missing endpoint for OTLP in NGINX #3479
```

Reply:
Changed